source/common/network/address_impl.cc

```diff
@@ -43,6 +43,13 @@ void validateIpv6Supported(const std::string& address) {
   }
 }
 
+// Returns the abstract path with embedded nulls replaces with '@'.
+std::string friendlyNameFromAbstractPath(const char* path, size_t num) {
```

Comment:
use absl::string_view as parameter instead of const char* and size_t pair?

Reply:
I chose that way only because I still need to explicitly pass the length into a string_view parameter since the abstract path has embedded nulls. 

To avoid copies I could also pass the `friendly_name_` into the function as a reference to avoid copy constructing the return value?